src/resources/scepter.ts

```diff
@@ -0,0 +1,188 @@
+import {
+  $effect,
+  $familiar,
+  $item,
+  $items,
+  $location,
+  $skill,
+  $slot,
+  AugustScepter,
+  clamp,
+  get,
+  getModifier,
+  have,
+} from "libram";
+import { globalOptions } from "../config";
+import { embezzlerCount } from "../embezzler";
+import { EMBEZZLER_MULTIPLIER } from "../lib";
+import { Potion } from "../potions";
+import { garboAverageValue, garboValue } from "../value";
+import { canAdventure, canEquip, Item, myLevel, myMeat, Skill, toSlot, useSkill } from "kolmafia";
+
+type ScepterSkill = {
+  skill: Skill;
+  value: () => number;
+  type: "special" | "summon" | "buff";
+};
+const SKILL_OPTIONS: ScepterSkill[] = [
+  // August 1 deliberately omitted; does not trigger on monster replacers
+  {
+    skill: $skill`Aug. 2nd: Find an Eleven-Leaf Clover Day`,
+    value: () =>
+      canAdventure($location`Cobb's Knob Treasury`)
+        ? EMBEZZLER_MULTIPLIER() * get("valueOfAdventure")
+        : 0,
+    type: "special",
+  },
+  {
+    skill: $skill`Aug. 3rd: Watermelon Day!`,
+    value: () => 3 * garboValue($item`watermelon`),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 4th: Water Balloon Day!`,
+    value: () => 3 * garboValue($item`water balloon`),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 5th: Oyster Day!`,
+    value: () =>
+      3 *
+      garboAverageValue(
+        ...$items`brilliant oyster egg, gleaming oyster egg, glistening oyster egg, lustrous oyster egg, magnificent oyster egg, pearlescent oyster egg, scintillating oyster egg`,
+      ),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 7th: Lighthouse Day!`,
+    value: () =>
+      new Potion($item`august scepter`, {
+        effect: $effect`Incredibly Well Lit`,
+        duration: 30,
+      }).gross(embezzlerCount()), // TODO: Yachtzee
+    type: "buff",
+  },
+  {
+    skill: $skill`Aug. 13th: Left/Off Hander's Day!`,
+    value: () =>
+      new Potion($item`august scepter`, {
+        effect: $effect`Offhand Remarkable`,
+        duration: 30,
+        effectValues: { meatDrop: 80 }, // Half a purse
+      }).gross(embezzlerCount()) +
+      (globalOptions.ascend
+        ? 0
+        : (5 + (have($familiar`Left-Hand Man`) ? 5 : 0)) * get("valueOfAdventure")),
+    type: "special", // Don't want to cast right away
+  },
+  {
+    skill: $skill`Aug. 14th: Financial Awareness  Day!`,
+    value: () => Math.min(100 * myLevel(), 1500, myMeat()) / 2,
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 16th: Roller Coaster Day!`,
+    value: () => 8 * get("valueOfAdventure"),
+    type: "special",
+  },
+  {
+    skill: $skill`Aug. 18th: Serendipity Day!`,
+    value: () => 1000, // Dummy value; we should some day calculate this based on free fight count, careful to avoid circular imports
```

Comment:
1000 is a very low total. on average you get an item with an autosell value of 50-100 meat per fight you have serendipity on for. even if you assume zero free fights, that's 1500-3000 (or 1750-3500 for PYEC-havers); but for high shiny accounts that is +70ish free fights for like 5000-10000 value. i'm comfortable with just saying the free fight tabulation is out of scope but i think it's probably worth ticking this up to at least 3000, assuming it's worth 100 per turn.